packages/frontend/src/hooks/chat/useMessage.ts

```diff
@@ -139,6 +140,9 @@ export default function useMessage() {
         ...message,
       })
 
+      // Notify about the sent message (listeners can filter by message type if needed)
+      notifyWebxdcMessageSent(accountId, chatId, message)
```

Comment:
TBH I'd prefer the previous "generic" approach, but with only a comment warning against using these functions.
But alright.